src/shared/sam/packagers/samPackager.ts

```diff
@@ -81,7 +81,7 @@ export class SamPackager {
     private async validateFoldersExist(): Promise<void> {
         const missingFolders: boolean[] = (await Promise.all(
             Object.keys(this._folderMap)
-                .map(folder => SystemUtilities.fileExists(folder))
```

Comment:
Do you also need to `await` the result of `fileExists`?

Reply:
No, I'm returning them all, then calling Promise.all